monitor/pom.xml

```diff
@@ -71,6 +73,51 @@
           <target>1.7</target>
         </configuration>
       </plugin>
+
+			<plugin>
+        		<artifactId>maven-antrun-plugin</artifactId>
+        		<version>1.6</version>
+        		<executions>
+          			<execution>
+            			<phase>clean</phase>
```

Comment:
This task should not be linked to maven `clean` phase. Please change this linking (and the one just below) to `generate-sources`.